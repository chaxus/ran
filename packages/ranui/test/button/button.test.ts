import Button from '@/components/button'

describe('button', () => {
  it('init button function', () => {
    const element = new Button()
    expect(element).toBeTruthy()
  })
  it('test button ui', () => {
    new Button()
    const rButton = document.createElement('r-button')
    document.body.appendChild(rButton)
    expect(document.body).toMatchSnapshot()
  })
})
