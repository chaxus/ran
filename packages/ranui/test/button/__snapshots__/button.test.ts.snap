// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button test button ui 1`] = `
<body>
  <r-button />
</body>
`;
